lib/plugins/orionUrls.js

```diff
@@ -29,68 +29,86 @@
  * assigned.
  */
 module.exports = [
-  /* Standard NGSI operations */
```

Comment:
Yes, I corrected the indentation in this file, but all the changes are bellow the `V2 Operations` tag.